bpf/kmesh/workload/include/authz.h

```diff
@@ -22,6 +23,25 @@ struct {
     __uint(max_entries, MAP_SIZE_OF_AUTH_POLICY);
 } map_of_authz SEC(".maps");
 
+struct match_ctx {
+    __u32 action;
+    __u8 policy_index;
+    __u8 n_rules;
+    wl_policies_v *policies;
+    void *rulesPtr;
+};
+
+/*
+ * This map is used to store the variable that
+ * xdp_auth needs to pass during the tail call
+ */
+struct {
+    __uint(type, BPF_MAP_TYPE_HASH);
+    __uint(key_size, sizeof(struct bpf_sock_tuple));
+    __uint(value_size, sizeof(struct match_ctx));
+    __uint(max_entries, 256);
+} tailcall_info_map SEC(".maps");
```

Comment:
suggest add `kmesh` prefix

Add     __uint(map_flags, BPF_F_NO_PREALLOC);

Is 256 enough?  I think we should greatly increase to support higher throughput  if BPF_F_NO_PREALLOC set